### PR TITLE
chore: remove EOL Camel Quarkus releases.

### PR DESCRIPTION
**Pull request by davsclaus**

### CODE REVIEW

**Comment by davsclaus** [comment]
Can we removed CQ 2.7.x release (its listed as EOL in Aug 2022)
https://camel.apache.org/camel-quarkus/2.14.x/

**Review by oscerd — APPROVED** [review]

**Comment by davsclaus** [comment]
Blog posts to be fixed

[check:links    ] Found invalid urls in blog/2022/07/camel-quarkus-release-2.11.0/index.html:
[check:links    ] 	Linked file at path camel-quarkus/2.11.x/reference/extensions/hashicorp-vault.html does not exist!
[check:links    ] 	Linked file at path camel-quarkus/2.11.x/reference/extensions/jq.html does not exist!
[check:links    ] 	Linked file at path camel-quarkus/2.11.x/reference/extensions/dataset.html does not exist!
[check:links    ] 	Linked file at path camel-quarkus/2.11.x/migration-guide/2.11.0.html does not exist!
[check:links    ] Found invalid urls in blog/2022/06/camel-quarkus-release-2.10.0/index.html:
[check:links    ] 	Linked file at path camel-quarkus/2.10.x/reference/extensions/azure-key-vault.html does not exist!
[check:links    ] 	Linked file at path camel-quarkus/2.10.x/migration-guide/2.10.0.html does not exist!
[check:links    ] 	Linked file at path camel-quarkus/2.10.x/reference/extensions/datasonnet.html does not exist!
[check:links    ] Found invalid urls in blog/2022/09/camel-quarkus-release-2.12.0/index.html:
[check:links    ] 	Linked file at path camel-quarkus/2.12.x/reference/extensions/cxf-soap.html does not exist!
[check:links    ] 	Linked file at path camel-quarkus/2.12.x/user-guide/testing.html does not exist!
[check:links    ] Found invalid urls in blog/2022/04/camel-quarkus-release-2.[8](https://github.com/apache/camel-website/actions/runs/3767216103/jobs/6404534022#step:5:9).0/index.html:
[check:links    ] 	Linked file at path camel-quarkus/2.8.x/migration-guide/2.8.0.html does not exist!

**Comment by ppalaga** [comment]
Thanks @davsclaus for spotting this. Yes, I would say we can remove 2.7 now that we have 2.13. On the same note, 2.13 should be marked as LTS till we have another LTS on top of Quarkus 3.x and Camel 4.x. Hard to say when exactly it will be - maybe April 2023 is safe enough?

**Comment by zbendhiba** [comment]
it seems there's a problem, we don't see the last release 2.15 in the website

File: antora-playbook-snippets/antora-playbook.yml [hunk 1]
```diff
@@ -60,20 +59,13 @@ content:
         - main
         - 0.10.x
         - 0.9.x
-        - 0.7.x
       start_path: docs
 
     - url: https://github.com/apache/camel-quarkus.git
       branches:
         - main
         - 2.14.x
```
Comment (zbendhiba) [inline antora-playbook-snippets/antora-playbook.yml:67]: @davsclaus We could add 2.15.x, as I forgot to do so.

Comment (davsclaus) [reply antora-playbook-snippets/antora-playbook.yml:68]: Yes I am adding it

**Review by zbendhiba — APPROVED** [review]
thanks @davsclaus

**Pull request merged**